bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/template/RuleTemplate.java

```diff
@@ -88,12 +87,13 @@
     private final Visibility visibility;
 
     /**
-     * This field defines a set of configuration properties of the future {@link Rule} instances.
+     * This field defines a set of configuration properties of the future {@link RuleImpl} instances.
```

Comment:
Rule vs RuleImpl (see above)...
I don't comment this anymore, please have a look at all places.